detection:
  ros__parameters:
    conf_threshold: 0.5    # Confidence threshold for detection
    nms_threshold: 0.4     # NMS threshold for YOLO
    topics:
      image_in: "/image_in"     # Input image topic
      image_out: "/image_out"   # Output image topic
      detected_human: "/detected_human"  # Human detection result topic
    rcv_timeout_secs: 1.0
    angular_chase_multiplier: 0.7
    forward_chase_speed: 0.1
    search_angular_speed: 0.5
    max_size_thresh: 1
    filter_value: 0.9
